src/api-fetch-setup.js

```diff
@@ -18,6 +14,9 @@ const fetchHandler = ( options ) => {
 	const responsePromise = fetchRequest( options.path );
 
 	const parseResponse = ( response ) => {
+		if ( typeof response === 'string' ) {
+			response = JSON.parse( response );
```

Comment:
For Android, it is a fair bit easier to just send a string across the bridge instead of a json object. Let me know if there's a reason we should prefer an object though. I can definitely do it, it just requires a fair bit of code something along [these lines](https://github.com/AppsFlyerSDK/react-native-appsflyer/blob/master/android/src/main/java/com/appsflyer/reactnative/RNUtil.java).